src/reconcile/rec_write.c

```diff
@@ -2417,9 +2420,18 @@ __rec_split_write(WT_SESSION_IMPL *session, WT_RECONCILE *r, WT_REC_CHUNK *chunk
         /* We must only have one delta. Building deltas for split case is a future thing. */
         WT_ASSERT(session, last_block);
         multi->block_meta = *block_meta;
```

Comment:
I see we are not setting `base_checkpoint_id`; is this because we are currently building a delta, so we are keeping the original page's checkpoint ID? If so, can you please add the comment?

Reply:
Yes, that's correct. Will do

Reply:
done